.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,189 @@
+name: BENCHMARK
+on:
+  workflow_dispatch:
+    inputs:
+      cpus:
+        description: 'docker cpus '
+        default: 2
+        require: true
+      memory:
+        description: 'docker memory'
+        default: '4g'
+        require: true
+      benchmark-time:
+        description: 'Benchmark time(h/m/s)'
+        default: 30m
+        require: true
+      indexerVersion:
+        description: 'indexer version'
+        default: 'latest'
+        require: true
+      deployment:
+        description: 'Deployment'
+        default: 'QmakYyTWzTxaDU842CczyDWV51Ay9E55k7kbRRXTB3f395'
```

Comment:
need to a near project, and also near endpoint

Reply:
should I deploy a starter project?